nrf52-hal-common/src/twim.rs

```diff
@@ -372,6 +372,31 @@ impl<T> Twim<T> where T: TwimExt {
     }
 }
 
+/// Implementation of embedded_hal::blocking::i2c Traits
+
+impl<T> embedded_hal::blocking::i2c::Write for Twim<T> where T: TwimExt {
+    type Error = Error;
+
+    fn write<'w>(&mut self, addr: u8, bytes: &'w [u8]) -> Result<(), Error> {
+        self.write(addr, bytes)
+    }
+}
+
```

Comment:
Just had a quick glance at the code, but is it not possible to add the implements traits- clause to existing functions? It looks like they already have the correct signature.

Reply:
The implemented functions are part of the default implementation of the `Twim` trait. We need to implement multiple new traits. My intuition tells me the correct way of doing so is by moving the implementation details of the function to the trait implementation. This however is likely to break existing implementations using the library as now the code will require importing the Trait for the implementation to be available. So I've done a call to the existing implementation instead.

I do not completely understand your question, can you be more explicit with what you mean by `add the implement traits clause to existing functions`?

Reply:
I think it's ok to implement it like this for now and change it later for exactly the reason of possible breakage.
What do you think @jamesmunns @hannobraun ?

```
I do not completely understand your question, can you be more explicit with what do you mean by add the implement traits clause to existing functions?
```
I think by that he simply meant what you just described, impl the Trait instead of just functions.

Reply:
> I think it's ok to implement it like this for now and change it later for exactly the reason of possible breakage.
> What do you think @jamesmunns @hannobraun ?

I agree. I think having the two method versions, as done here, is probably the better option in general, although I don't feel strongly about this.

In any case, I'm fine with merging now (assuming the reviewers are happy; I haven't fully reviewed). We can then decide to leave it as is or change it at a later point.